composer.json

```diff
@@ -14,22 +14,24 @@
     "require": {
         "php": ">=7.2",
         "doctrine/doctrine-bundle": "^1.8|^2.0",
-        "doctrine/orm": "^2.6",
+        "doctrine/orm": "^2.7",
         "league/oauth2-server": "^8.0",
         "psr/http-factory": "^1.0",
-        "sensio/framework-extra-bundle": "^5.3",
-        "symfony/framework-bundle": "^3.4|^4.2",
-        "symfony/psr-http-message-bridge": "^1.2",
-        "symfony/security-bundle": "^3.4|^4.2"
+        "sensio/framework-extra-bundle": "^5.5",
+        "symfony/psr-http-message-bridge": "^1.3",
+        "symfony/security-bundle": "^4.4|^5.0",
+        "symfony/framework-bundle": "^4.4|^5.0"
     },
     "require-dev": {
         "ext-timecop": "*",
         "ext-xdebug": "*",
-        "nyholm/psr7": "^1.1",
-        "phpunit/phpunit": "^8.4",
-        "symfony/browser-kit": "^3.4|^4.2",
-        "symfony/phpunit-bridge": "^3.4|^4.4",
-        "zendframework/zend-diactoros": "^2.1"
+        "friendsofphp/php-cs-fixer": "^2.16",
```

Comment:
@elchris This was wrongly rebased. We removed CS fixer as a composer dev dependency as it is now a part of the Docker image as a PHAR.

Reply:
ah sorry about that, fixed now, thank you!